install_config/configuring_local.adoc

```diff
@@ -189,15 +200,18 @@ provisioner: kubernetes.io/no-provisioner
 volumeBindingMode: WaitForFirstConsumer
 ----
 +
-See the link:https://raw.githubusercontent.com/openshift/origin/master/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml[template] for other configurable options. This template creates a DaemonSet that runs a
-pod on every node. The pod watches directories specified in the `ConfigMap` and
+See the link:https://raw.githubusercontent.com/openshift/origin/master/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml[local storage provisioner template] for other configurable options. This template creates a DaemonSet that runs a
+pod on every node. The pod watches directories specified in the ConfigMap and
 creates PVs for them automatically.
 +
-The provisioner runs as root to be able to clean up the directories when a PV is released and all data needs to be removed.
+The provisioner runs as root to clean up the directories when a PV is released and all data needs to be removed.
```

Comment:
Maybe "The provisioner runs with root permissions because it removes all data from the modified (or a more appropriate adjective of your choice) directories when a PV is released."